esmvaltool/utils/cmorizers/obs/cmor_config/ESACCI-OC.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: ESACCI-OC
+  version: 'fv3.1'
+  tier: 2
+  modeling_realm: sat
+  project_id: CMIP5
```

Comment:
```suggestion
  project_id: OBS
```

Reply:
Wasn't the CMIP5 table the reference for all cmorizers or something has changed?

Reply:
Only the non-derived ones.

Reply:
@mattiarighi  So we have to keep CMIP5? 😅

Reply:
See this [discussion](https://github.com/ESMValGroup/ESMValTool/pull/1056/files/73e54036d89648851a5362055477fe698c66aff0#pullrequestreview-234997950).

Reply:
The present data for `chl` are not derived, but available on both CMIP5 and OBS tables.
I tested the cmorizer with both tables and it works fine.
Personally I would prefer to stick to CMIP5 one but I leave to you the final decision.

Reply:
At the moment the OBS table is the same as CMIP5 + custom. However, since this is observational data and not model data, the correct project here is OBS, same as in the recipe that uses this data.